### ISSUE TITLE
FR: inline ignore comments

### ISSUE THREAD

**Issue by chrisgrieser** [elijah-potter/harper#105]
In some cases, it might be useful to simply disable harper for a paragraph or a line. Most linters/LSPs allow for inline directives via comments, e.g. `---@diagnostic disable-next-line: {rulename}` in the Lua LSP.

**Comment by bassamsdata** [comment]
It's true and a very useful feature.

It would also be helpful to add an inline directive for specific chunks, similar to how it's done in StyLua. For example:
```lua
-- stylua: ignore start 
-- whatever code or comments should be ignored by the linter
-- stylua: ignore end 
```

Additionally, to ignore an entire file, you can place `-- stylua: ignore start` at the beginning of the file.

Thank you

**Comment by elijah-potter** [comment]
This is similar to #102. While I like the stylua comments, I think the format they propose is much better (and plays nicer with JavaDoc's HTML and JsDoc's Markdown.

For reference:

```
<!-- markdownlint-disable-file MD033 -->
```

**Comment by chrisgrieser** [comment]
The recently added "ignore lint" goes in a somewhat similar direction, however it does not add an ignore comment, it only temporarily ignores the diagnostic for the current session. Until ignore comments are added, this appears to be the best solution for ignoring things. (Just commenting this for other people finding this issue.)

**Comment by Isaac7777-cpu** [comment]
May I please ask what is the "ignore lint" option and how should I use it?

**Comment by elijah-potter** [comment]
When you're using the Harper language server (in VS Code, Neovim, Helix, etc.), you'll get a Code Action that says something like "Ignore suggestion". This tell Harper to ignore that error for the remainder of the session. In future, I'll add persistence (like which already exists in the Obsidian and WordPress plugins).

That said, we have a PR for instructing Harper to ignore comment blocks: https://github.com/Automattic/harper/pull/861

**Comment by Isaac7777-cpu** [comment]
Yep, I think I see that as well! Thank you very much (I am using nvim).

**Comment by FlawlessCasual17** [comment]
@elijah-potter Can you please add these inline ignore comments (from PR #861) to the documentation? It would be extremely helpful.
